app/views/moderations/_mod_sidebar_left.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="crayons-layout__sidebar-left">
+  <p class="fw-heavy fs-3xl mod-center-heading">Mod Center</p>
+  <div class="fw-bold fs-s mod-center-section-title">
+    INBOX
```

Comment:
Very minor, but I've noticed a lot of our text labeling is steering away from all caps to just the first letter being capitalized.